osu.Game/Overlays/Mods/ModSearchContainer.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Overlays.Mods;
+
+public partial class ModSearchContainer : SearchContainer
+{
+    /// <summary>
+    /// A string that should match the <see cref="IFilterable"/> children
+    /// </summary>
+    public string ForcedSearchTerm
```

Comment:
This workaround is still here? Didn't I say that it was a bad hack and it would be probably a better idea to try adding a method to force refiltering framework-side first [here](https://github.com/ppy/osu/discussions/22893#discussioncomment-5790196)?

Reply:
Sorry, I misread it. I'll PR this as soon as I have time. Also wouldn't it be better to add protected method to force refiltering?

Reply:
Whichever works.